app/templates/service_update_audits.html

```diff
@@ -0,0 +1,154 @@
+{% from 'macros/page_heading.html' import page_heading %}
+
+{% extends "_base_page.html" %}
+{% block page_title %}
+Digital Marketplace admin
+{% endblock %}
```

Comment:
Do you think we should set this in `_base_page.html`? It looks like it would be the same title for all pages in the admin app.